draft-age-masque-connect-ip.md

```diff
@@ -260,107 +261,74 @@ the peer of the assignment.
 
 The ROUTE_ADVERTISEMENT capsule (see {{iana-types}} for the value of the
 capsule type) allows an endpoint to communicate to its peer that it is willing
-to route traffic to a given prefix. This indicates that the sender has an
-existing route to the prefix, and notifies its peer that if the receiver of the
-ROUTE_ADVERTISEMENT capsule sends IP packets for this prefix in HTTP Datagrams,
-the sender of the capsule will forward them along its preexisting route. Any of
-these addresses can be used as the destination address on IP packets originated
-by the receiver of this capsule.
+to route traffic to a set of IP address ranges. This indicates that the sender
+has an existing route to each address range, and notifies its peer that if the
+receiver of the ROUTE_ADVERTISEMENT capsule sends IP packets for one of these
+ranges in HTTP Datagrams, the sender of the capsule will forward them along its
+preexisting route. Any address which is in one of the address ranges can be
+used as the destination address on IP packets originated by the receiver of
+this capsule.
 
 ~~~
 ROUTE_ADVERTISEMENT Capsule {
   Type (i) = ROUTE_ADVERTISEMENT,
   Length (i),
+  IP Address Range (..),
```

Comment:
I assume the idea here is to have multiple IP Address Range fields in one capsule? I believe you need to use [] for a vector then.